src/docs/sphinx/user_guide/simple_conduction_tutorial.rst

```diff
@@ -0,0 +1,292 @@
+building a rectangular mesh, we extract the corresponding part of the options variant and use that to construct
+the mesh.  The Lua representation is as follows:
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _mesh_start
+   :end-before: _mesh_end
+   :language: Lua
+
+Constructing the Physics Module
+-------------------------------
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _create_module_start
+   :end-before: _create_module_end
+   :language: C++
+
+When using the C++ API, the ``ThermalConduction`` constructor requires the polynomial order of the elements and the
+solver options to be used when inverting the stiffness matrix, in addition to the mesh.  Since we're setting up
+a steady-state simulation, we can just use the ``defaultQuasistaticOptions``.
+
+Using Lua
+"""""""""
+
+Once the configuration options are deserialized from Inlet, we can use them to construct the ``ThermalConduction``
+object:
+
+.. literalinclude:: ../../../../examples/simple_conduction/with_input_file.cpp
+   :start-after: _create_module_start
+   :end-before: _create_module_end
+   :language: C++
+
+Unlike the C++-only version, we don't need to specify the order or the solver options in the constructor
+because they're in the input file:
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _solver_opts_start
+   :end-before: _solver_opts_end
+   :language: Lua
+
+Configuring the Physics Module
+------------------------------
+
+The following sections demonstrate a subset of the configuration options available with Serac's
+``ThermalConduction`` module.
+
+.. note::
+  The C++-only API requires method calls for configuration, while the Lua-based approach typically only requires changes
+  to the input file.  This is because the configuration options are all part of the ``ThermalConduction::InputOptions``
+  extracted from Inlet.
+
+Configuring Material Conductivity
+---------------------------------
+
+Instead of using a monolithic material model, the ``ThermalConduction`` module currently allows for material parameters
+like conductivity, specific heat capacity, and density to be configured individually.
+
+Using C++
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/without_input_file.cpp
+   :start-after: _conductivity_start
+   :end-before: _conductivity_end
+   :language: C++
+
+Using Lua
+"""""""""
+
+.. literalinclude:: ../../../../examples/simple_conduction/conduction.lua
+   :start-after: _conductivity_start
+   :end-before: _conductivity_end
+   :language: Lua
+
+Setting Thermal (Dirichlet) Boundary Conditions
+-----------------------------------------------
+
+The following snippets add a Dirichlet boundary condition that constrains the temperature to 1.0 at boundary
```

Comment:
Per our discussion in the meeting today, it would be more interesting to have a function-defined boundary condition (like T=x). I think this would showcase the capability a bit more while not increasing complexity.